build.gradle

```diff
@@ -57,8 +57,7 @@ jar {
 dependencies {
     api platform('com.exactpro.th2:bom:3.0.0')
 
-    implementation 'com.exactpro.th2:common:3.29.0'
-    implementation 'com.exactpro.th2:store-common:3.2.0'
+    implementation 'com.exactpro.th2:common:3.29.3-th2-2715-1481975477-SNAPSHOT'
```

Comment:
Use a release version instead of special build

Reply:
done